src/constructs/core/ssm.ts

```diff
@@ -0,0 +1,69 @@
+import type { IStringListParameter, IStringParameter } from "@aws-cdk/aws-ssm";
```

Comment:
Let's export this file in the [index](https://github.com/guardian/cdk/blob/main/src/constructs/core/index.ts) too?